save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt

```diff
@@ -65,8 +62,8 @@ class FallbackView : AbstractView<FallbackViewProps, State>(Style.SAVE_LIGHT) {
                     }
                 }
             } else {
-                a {
-                    href = "${window.location.origin}/"
```

Comment:
are you sure? Sounds like it was done intentionally

Reply:
Fixed